packages/dotcom-ui-footer/src/index.tsx

```diff
@@ -20,7 +21,10 @@ export function Footer(props: TFooterProps) {
   const footerData = props.data.footer.items
   const theme = props.theme ? `${props.theme}` : 'dark'
   return (
-    <footer id="site-footer" className={`o-footer o-footer--theme-${theme}`} data-o-component="o-footer">
+    <footer
+      id="site-footer"
+      className={`o-footer o-footer--theme-${theme} ${props.customClass}`}
```

Comment:
What happens in the event of a `customClass` attribute not being added to `props`?

It seems for `theme` that default values are set on lines 22 and 40 respectively and that maybe the same may be required here.

I may well be wrong but it looks like it might produce: `o-footer o-footer--theme-dark undefined`.

Reply:
I also wonder if it is possible to avoid trailing whitespace when `props.customClass` is absent (maybe by putting classes in an array and `.join`ing them with `' '`).